e2e-tests/src/auth/checkout.npg-payment-auth.integration.test.ts

```diff
@@ -54,19 +50,19 @@ describe("Checkout login and payment flow", () => {
         /*
          * 1. Payment with valid notice code
         */
-        const CODICE_AVVISO = String(VALID_CARD_DATA.fiscalCodePrefix) + Math.floor(Math.pow(10, 12) + Math.random() * 9 * Math.pow(10, 12))
-        //const CARD = CARD_TEST_DATA.cards[1];
+        const CARD = CARD_TEST_DATA.cards[6];
```

Comment:
I was thinking, is there card payment really needed?
We can perform an APM payment here, such as paypal.
This modification will make those tests faster (there is no input form card page load that has an external b.e. downstream dependency)